### ISSUE TITLE
Command line config & Icons

### ISSUE THREAD

**Issue by Timonzimm** [astrada/google-drive-ocamlfuse#589]
Hi,
1) I'm trying to mount a list of Team Drives from a python script and therefore would like to modify the configurations "on-the-fly", such as `team_drive_id`.

Is it somehow possible? So far, I thought about creating a `label` for each team drive, with the appropriate config file in each label folder, but it feels very hacky and cumbersome.

2) Follow-up question, how do you exactly define `document_icon`? Could you provide me with an example?
The documentation is lacking and all my trials so far were unsuccessful (icon stayed the same, even after clearing cache).

**Comment by astrada** [comment]
No, currently it isn't possibile, because changing the team drive id on the fly would invalidate the cache. 

> So far, I thought about creating a label for each team drive

This works.
